### ISSUE TITLE
Wheel for M1 apple?

### ISSUE THREAD

**Issue by pbelange** [hibtc/cpymad#118]
Hi, was there more work done in getting a wheel for MAC M1 (arm64 architectures)? I still get an error similar to the one of a previous issue:
```
src/cpymad/libmadx.c:777:10: fatal error: 'madX/mad_types_f.h' file not found
      #include "madX/mad_types_f.h"
```

**Comment by pbelange** [comment]
See 
---
Potentially solved with #114
_Originally posted by @ab-gh in https://github.com/hibtc/cpymad/issues/113#issuecomment-1266876127_

**Comment by coldfix** [comment]
I haven't tried it. Maybe @ab-gh knows more? Since I don't have access to a Mac, I'd rather let someone with more experience add a build for it. If no one steps up, I might look into it in a few weeks.

Closing as duplicate of #113.
